assets/js/components/Popup.js

```diff
@@ -319,7 +324,22 @@ module.exports = class Popup {
     if (this.isOpen) {
       this.positionPopupHitBox(e);
     } else {
+      this.popupHitBox.setAttribute('aria-expanded', 'false');
+      this.popupHitBox.removeAttribute('tabindex', -1);
+      this.popupHitBox.setAttribute('aria-hidden', 'true');
       this.popupHitBox.style.display = 'none';
+      this.$link.focus();
+    }
+
+    if (this.isOpen) {
+      this.popupHitBox.addEventListener('blur', () => {
+        this.popupHitBox.setAttribute('aria-expanded', 'false');
```

Comment:
There's a lot of duplication here with the `else` block defined on [line 326](https://github.com/elifesciences/pattern-library/pull/930/files#diff-752da1b00c0db6cd94aafc27aa81c946R326). Suggest extracting the duplication into a separate method.

Reply:
agree

Reply:
Completed